cluster/gce/gci/configure-helper.sh

```diff
@@ -480,6 +480,7 @@ function start-kubelet {
     if [[ ! -z "${KUBELET_APISERVER:-}" && ! -z "${KUBELET_CERT:-}" && ! -z "${KUBELET_KEY:-}" ]]; then
       flags+=" --api-servers=https://${KUBELET_APISERVER}"
       flags+=" --register-schedulable=false"
```

Comment:
the salt replaces this line with the new one, why leave both in the gci configs?